test_runner/fixtures/utils.py

```diff
@@ -57,6 +57,8 @@
 )
 # fmt: on
 
+USE_LFC = os.environ.get("USE_LFC") != "false"
```

Comment:
Is it also worth exposing as a pytest fixture?

Reply:
Wouldn't this run with LFC in a local environment by default? `None != "false"` would evaluate to true.

Reply:
This was done intentionally: if the environment variable is not set, we will run with LFC enabled.
I believe it is worth exposing because this logic is used in many parts of code.
But it is worth commenting, for sure.